app/src/main/java/com/courseproject/tindar/ds/DatabaseHelper.java

```diff
@@ -12,16 +12,20 @@
 import com.courseproject.tindar.usecases.editfilters.EditFiltersDsResponseModel;
 import com.courseproject.tindar.usecases.editprofile.EditProfileDsGateway;
 import com.courseproject.tindar.usecases.editprofile.EditProfileDsResponseModel;
+import com.courseproject.tindar.usecases.likelist.LikeListDsGateway;
 
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.Date;
 import java.util.GregorianCalendar;
 
-public class DatabaseHelper extends SQLiteOpenHelper implements EditProfileDsGateway, EditFiltersDsGateway {
+public class DatabaseHelper extends SQLiteOpenHelper implements EditProfileDsGateway, EditFiltersDsGateway,
+        LikeListDsGateway {
     private static DatabaseHelper dbInstance;
 
     private static final String TABLE_ACCOUNTS = "accounts";
+    private static final String TABLE_LIKES = "likes";
```

Comment:
This way of implementing this seems very strange to me. Wouldn't it make more sense to make a separate table for likes, where one column is the "liker" and the other is the "likee"? And something similar for the match list?